indico/modules/events/editing/templates/emails/editor_judgment_notification.txt

```diff
@@ -1,11 +1,16 @@
 {% extends 'events/editing/emails/base_email.txt' %}
 
-{% block subject -%}{% trans %}New editor judgment{% endtrans %}{%- endblock %}
+{% set title=contribution.verbose_title %}
+{% set revision_type = {3: "made changes to", 6: "requested changes to", 7: "accepted", 8: "rejected"}[revision_type] | default("left a judgment to") -%}
```

Comment:
concatenating sentences like this, while nicer for coding, really doesn't work for translations... even if the strings here were wrapped in i18n, it won't work for many languages which have different sentence structures
i'd suggest writing out the full title for each possible action (example in comment below)